### PR TITLE
AAD to Microsoft Entra updates

### PR DESCRIPTION
**Pull request by stevemunk**
There are no new or modified APIs in this PR, only changes to the documentation that are related to the AAD to Entra rebranding.

### CODE REVIEW

**Comment by stevemunk** [comment]
@tg-msft
@weidongxu-microsoft
@tjprescott

Please advise on what I need to do to get this PR reviewed and merged. The [Swagger ModelValidation](https://github.com/Azure/azure-rest-api-specs/pull/26725/checks?check_run_id=18800545898) errors have nothing to do with the changes that I've made, **_and they appear to be false negatives_**. There are two types of errors that I see:

### 1: RESPONSE_SCHEMA_NOT_IN_SPEC

> Response statusCode '{0}' for operation '{1}' has response body provided in the example, however the response does not have a "schema" defined in the swagger spec.

In call cases I've looked at, a response is defined. for example:

```
         "responses": {
          "200": {
            "$ref": "../../../Common/preview/1.0/common.json#/responses/200AsyncV2"
          }	
```
		  
If you look at what it is referencing in "../../../Common/preview/1.0/common.json#/responses/200AsyncV2":

```
     "200AsyncV2": {
      "description": "The operation is running or complete. If the operation was successful, use the Resource-Location header to obtain the path to the result.",
      "schema": {
        "$ref": "#/definitions/LongRunningOperationResult"
      }
```

You can look at what it is referencing in "#/definitions/LongRunningOperationResult" [here](https://github.com/Azure/azure-rest-api-specs/blob/4e068b4e8b2b3b754486a623222b28dbf95fcda1/specification/maps/data-plane/Common/preview/1.0/common.json#L12)

It appears that the response _does_ have a "schema" defined in the swagger spec, if not please advise.

### 2: INVALID_TYPE

The first invalid type error is referring to the bounding box property which is defined as an array of numbers (doubles) [see here](https://github.com/Azure/azure-rest-api-specs/blob/4e068b4e8b2b3b754486a623222b28dbf95fcda1/specification/maps/data-plane/Creator/preview/2.0/wfs.json#L92:16). This is correct.

In the example, it shows [this](https://github.com/Azure/azure-rest-api-specs/blob/4e068b4e8b2b3b754486a623222b28dbf95fcda1/specification/maps/data-plane/Creator/preview/2.0/examples/wfs/WFS_GetFeatures.json#L8:13): "bbox": "-122, 47, -120, 46" 

It appears that this is correct, if not please advise.

File: specification/maps/data-plane/Microsoft.Maps/Data/preview/2.0/data.json [hunk 1]
```diff
@@ -250,7 +250,7 @@
             }
           },
           "401": {
-            "description": "Access denied due to invalid subscription key or invalid Azure Active Directory (Azure AD) bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Azure AD bearer token.",
+            "description": "Access denied due to invalid subscription key or invalid  Microsoft Entra ID bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Microsoft Entra ID bearer token.",
```
Comment (yulinscottkang) [inline specification/maps/data-plane/Microsoft.Maps/Data/preview/2.0/data.json:253]: There's an extra space in front of "Microsoft Entra ID", same for the following lines: 318, 407, 475, 534

File: specification/maps/data-plane/Microsoft.Maps/Alias/preview/2.0/alias.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
       }
     },
     "401": {
-      "description": "Access denied due to invalid subscription key or invalid Azure Active Directory (Azure AD) bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Azure AD bearer token.",
+      "description": "Access denied due to invalid subscription key or invalid  Microsoft Entra ID bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Microsoft Entra ID bearer token.",
```
Comment (yulinscottkang) [inline specification/maps/data-plane/Microsoft.Maps/Alias/preview/2.0/alias.json:71]: There's an extra space in front of the first "Microsoft Entra ID". Same for all other files.

File: specification/maps/data-plane/Microsoft.Maps/Data/preview/2.0/data.json (hunk 1 above)
Comment (stevemunk) [reply specification/maps/data-plane/Microsoft.Maps/Data/preview/2.0/data.json]: fixed

File: specification/maps/data-plane/Microsoft.Maps/Alias/preview/2.0/alias.json (hunk 2 above)
Comment (stevemunk) [reply specification/maps/data-plane/Microsoft.Maps/Alias/preview/2.0/alias.json]: fixed

File: specification/maps/data-plane/readme.md [hunk 3]
```diff
@@ -42,22 +42,40 @@ directive:
...
+  - DataRegistry/stable/2023-06-01/dataregistry.json
+  - Geolocation/preview/1.0/geolocation.json
+  - Render/stable/2022-08-01/render.json
+  - Route/preview/1.0/route.json
+  - Search/stable/2023-06-01/search.json
+  - Spatial/stable/2022-08-01/spatial.json
+  - Timezone/preview/1.0/timezone.json
+  - Traffic/preview/1.0/traffic.json
+  - Weather/stable/1.1/weather.json
+```
```
Comment (weidongxu-microsoft) [inline specification/maps/data-plane/readme.md:70]: I am not sure whether service wish to add a "stable" tag for multiple preview Swaggers.

Comment (stevemunk) [reply specification/maps/data-plane/readme.md:70]: @weidongxu-microsoft These changes were already approved and checked in. See [here](https://github.com/stevemunk/azure-rest-api-specs/commit/1d5886bb0ffec4abcf7cea5899aa77a8eeed8a89) for the commit.

Comment (weidongxu-microsoft) [reply specification/maps/data-plane/readme.md:70]: OK. If this already in, I am fine with it.

Just confused why it shows up in the diff.

File: specification/maps/data-plane/readme.md [hunk 4]
```diff
@@ -68,7 +86,6 @@ input-file:
   - Microsoft.Maps/Timezone/preview/1.0/timezone.json
   - Microsoft.Maps/Traffic/preview/1.0/traffic.json
   - Microsoft.Maps/Weather/preview/1.0/weather.json
-  - Microsoft.Maps/WFS/preview/2.0/wfs.json
```
Comment (scbedd) [inline specification/maps/data-plane/readme.md:71]: I believe this removal is why you're seeing the sudden errors for `unreferenced json from readme`. [the first avocado error](https://github.com/Azure/azure-rest-api-specs/pull/26725/checks?check_run_id=19202070457) specifically. I think it's iffy reporting, but all stemming from the unreferenced `wfs.json`. I'm betting it's breaking `oav`'s logic when resolving the swagger itself, which is why the examples are also throwing.

Comment (scbedd) [reply specification/maps/data-plane/readme.md:71]: I think if you make changes to the readme like this, you're going to need to go through the ARM review process as described in the PR body. (the image with labeling instructions etc in the automatically created body of the PR).

File: specification/maps/data-plane/readme.md (hunk 3 above)
Comment (scbedd) [reply specification/maps/data-plane/readme.md:70]: I'm also a bit confused that it's showing up on the diff.

**Review by tjprescott — APPROVED** [review]

**Review by tjprescott — CHANGES_REQUESTED** [review]
Approved in error and GH won't let me dismiss

**Comment by stevemunk** [comment]
@tjprescott, @scbedd, @weidongxu-microsoft: 99.9% of all errors have been addressed, that leaves us with one error. It is a [PoliCheck error](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3335298&view=logs&j=7765bc23-5042-5e9c-3803-46283e867697&t=be387a88-9677-5a75-c75d-006ece3d825d):

image

Pretty sure these three letters are not used anywhere in this swagger file (or any other Azure Maps swagger files). I looked in [SuccessfulWorldCopyrightRequest.json ](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Render/preview/2.1/examples/SuccessfulWorldCopyrightRequest.json)and couldn't find those three letters together except when part of 'Istituto' and 'constitute', and the following URL:

image

Any suggestions?

**Comment by tjprescott** [comment]
@stevemunk the blocking issue wasn't the policheck failure--it was the need to review and approve the suppression statements you added.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**